go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/golang/protobuf v1.2.0
 	github.com/google/go-cmp v0.2.0 // indirect
 	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
-	github.com/grpc-ecosystem/grpc-gateway v1.4.1
+	github.com/grpc-ecosystem/grpc-gateway v1.6.4
```

Comment:
I don't think we use the grpc-gateway stuff anymore? Can we get rid of it?

Reply:
I'd be super happy to get rid of it :)